src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {

```

Comment:
FigureSupplier should have two public methods(`getRandomFigure(), getDefaultFigure()`), others methods should be private.
Methods used in the inner of the class must be private
All private methods should be below public methods